Makefile

```diff
@@ -65,6 +65,7 @@ guess_platform_script := \
 guess_platform := $(shell $(guess_platform_script))
 
 all: all-$(guess_platform)
+check: check-$(guess_platform)
 
```

Comment:
Hmm, that sounds like a pretty sensible idea to me.  Please feel free to tweak stuff as appropriate. :+1: